content/en/blog/_posts/2021-03-09-Defining-NetworkPolicy-Conformance-For-CNIs

```diff
@@ -0,0 +1,373 @@
+---
+layout: blog
+title: "Defining NetworkPolicy Conformance for CNI providers"
+date: 2021-03-21
+slug: defining-networkpolicy-conformance-cni-providers
+---
+
+Authors: Matt Fennwick, Jay Vyas, Ricardo Katz, Amim Knabben, Douglas Landgraf
+
+A brief conversation around "node local" network policies in April of 2020 inspired the creation of a NetworkPolicy subproject from sig-network.
+
+In this post we'll discuss:
+
+- Why we created a Subproject for Network Policies 
+- How we changed the Kubernetes e2e framework to `visualize` NetworkPolicy implementation of your CNI provider
+- The initial results of our comprehensive NetworkPolicy Conformance validator, `Cyclonus`, built around these principles
```

Comment:
nit:
```suggestion
- The initial results of our comprehensive NetworkPolicy conformance validator, _Cyclonus_, built around these principles
```
(unless people literally type `Cyclonus` into a command line to run validation)

Reply:
People type cyclonus, here:
https://github.com/K8sbykeshed/cyclonus-artifacts/blob/main/jobs/antrea/cyclonus-job.yaml#L11